### PR TITLE
Add my location button to map.

### PR DESCRIPTION
**Pull request by hotchemi**
## What

Impl #47 .

## Change

- Add my current position button to map fragment.
- Add permission control code because it uses `ACCESS_FINE_LOCATION`
  - This code is just the show case of PermissionsDispatcher:D

## Screenshot

hammerheadmpa44ihotchemi02122016025044

hammerheadmpa44ihotchemi02122016025028

hammerheadmpa44ihotchemi02122016025012

### CODE REVIEW

**Comment by konifar** [comment]
Awesome...!!

File: app/src/main/java/io/github/droidkaigi/confsched/fragment/MapFragment.java [hunk 1]
```diff
@@ -53,23 +66,12 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
...
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
         binding = FragmentMapBinding.inflate(inflater, container, false);
-        initGoogleMap();
+        initGoogleMapWithCheck(this);
         setHasOptionsMenu(true);
         initBackPressed();
         return binding.getRoot();
     }
 
-    private void initBackPressed() {
```
Comment (hotchemi) [inline app/src/main/java/io/github/droidkaigi/confsched/fragment/MapFragment.java]: This is moved to below.

File: app/src/main/res/values-ar/strings.xml [hunk 2]
```diff
@@ -38,6 +38,11 @@
     <string name="map_party_name">مكان تجمع اجتماعي</string>
     <string name="map_party_building">معهد طوكيو للتكنولوجيا Kuramae كايكان (Kuramae هول)</string>
     <string name="map_address">معهد طوكيو للتكنولوجيا Ookayama الحرم الجامعي \ ن ميغورو-كو، طوكيو Ookayama 2&#8211;12&#8211;1</string>
+    <string name="map_fine_location_denied">النظر في منح ذلك من أجل الحصول على الموقع الحالي</string>
```
Comment (hotchemi) [inline app/src/main/res/values-ar/strings.xml]: Using google translation

**Comment by hotchemi** [comment]
Can I merge it?

**Comment by konifar** [comment]
LGTM! You can merge it.

**Comment by hotchemi** [comment]
Thx!!!!!!!

**Pull request merged**